src/core_plugins/i18n/server/i18n/index.js

```diff
@@ -0,0 +1,35 @@
+import  i18n from './i18n';
```

Comment:
No longer needed as `src/core_plugins/i18n/server/i18n/index.js ` is now obsolete as exported functions in `src/core_plugins/i18n/server/i18n/i18n.js ` are external.

Documentation added to `src/core_plugins/i18n/server/i18n/i18n.js `.